wysadzulice/migrations/0001_initial.py

```diff
@@ -13,31 +13,27 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Campaign',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('id', models.AutoField(primary_key=True, auto_created=True, serialize=False, verbose_name='ID')),
```

Comment:
As I wrote on gitter - there's is no need to review `migrations` directory (at least not uusally). These files are created automatically using: ```python manage.py makemigrations```.

For more informations: https://docs.djangoproject.com/es/1.9/topics/migrations/